packages/sdks/typescript/src/utils/syncRun.ts

```diff
@@ -64,7 +64,7 @@ export async function syncRun(
     })
 
     onError?.(error)
-    return !onError ? Promise.reject(error) : Promise.resolve()
+    return !onError ? Promise.reject(error) : Promise.resolve(undefined)
```

Comment:
bug, without this undefined we were returning type void type which ain't correct

Reply:
But was causing errors? Just curious

Reply:
it causes typescript static errors

Reply:
👍